test/clients/inst_prepdisk_test.rb

```diff
@@ -39,17 +39,36 @@
       allow(storage).to receive(:prepend_rootprefix).with("/proc").and_return "/dest/proc"
       allow(storage).to receive(:prepend_rootprefix).with("/sys").and_return "/dest/sys"
       allow(Yast::SCR).to receive(:Execute).and_return(true)
+      allow(Yast::Mode).to receive(:update).and_return(mode == :update)
     end
 
-    it "uses the destination directory to mount and prepare the result" do
-      expect(storage).to receive(:rootprefix=).with("/dest")
-      client.run
+    context "in normal mode" do
+      let(:mode) { :normal }
```

Comment:
Not sure if checking for `normal` mode makes much sense for this client. I would have expected `installation` and `update` to be tested, but `normal`...